config/config_new_default.yml

```diff
@@ -17,9 +17,15 @@ aprs:
     telem_timeout: 70
     message_spacing: 1
 eps:
+    looptime: 20
```

Comment:
All changes should be made directly into `config_default.yml`. In fact, feel free to delete `config_new_default.yml`

Reply:
done